### DISCUSSION TITLE
Turn on or off more relays with one command mqtt?

### DISCUSSION THREAD

**Discussion by Smoeller0** [arendst/Tasmota#17264] [Q&A] [answered]
Is there any solution to turn on or off 4 relays with one command?
I have 4 relays and wish i can send something like 1:on, 2:off .... but with one mqtt message.

is this possible?

**Comment by sfromis** [comment, marked as answer]
You can use the command `Power0` to turn on/off all relays at the same time.
Or you can use the `Backlog` command (in the topic) having a semicolon-delimited list (in the payload) of individual commands, including the Power command for whatever relays.
